js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
```

Comment:
Am I misunderstanding what you mean with
> `getMove` is unavailable (or, to be exact, provides no useful information) in replays.

then, since it's usage here is the same as before? 

Though, I probably shouldn't overwrite `move` when I look at it now.